test/rule-matches/aria-allowed-role-matches.js

```diff
@@ -1,47 +1,27 @@
 describe('aria-allowed-role-matches', function() {
   'use strict';
 
-  var fixture = document.getElementById('fixture');
-  var flatTreeSetup = axe.testUtils.flatTreeSetup;
+  var queryFixture = axe.testUtils.queryFixture;
   var rule;
 
   beforeEach(function() {
     rule = axe.utils.getRule('aria-allowed-role');
   });
 
-  afterEach(function() {
-    fixture.innerHTML = '';
-  });
```

Comment:
Why did you remove this?

Reply:
Our testutils file does this for us https://github.com/dequelabs/axe-core/blob/develop/test/testutils.js#L440-L442